### ISSUE TITLE
Autoprefixer removing certain prefixes.

### ISSUE THREAD

**Issue by Vidhya-Dilip** [postcss/autoprefixer#1220]
Autoprefixer is removing ' -webkit-box-orient: vertical' css property while on production.

What I use:
.limit-text {
  overflow: hidden;
  text-overflow: ellipsis;
  display: -webkit-box;
  -webkit-box-orient: vertical;
  -webkit-line-clamp: 2;
}

What is seen on production:
.limit-text {
  overflow: hidden;
  text-overflow: ellipsis;
  display: -webkit-box;
  -webkit-line-clamp: 2;
}

This causes issue in UI.
I'm using autoprefixer with Vue 2.5.2.

**Comment by ai** [comment]
Use:

```css
  /* autoprefixer ignore next */
  -webkit-box-orient: vertical;
```

Does this code works in Firefox and other non-webkit browsers?

**Comment by mgol-sumo** [comment]
@ai Firefox has implemented [`webkit-line-clamp`](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp) in v68 (currently in beta). To use it, you have to set `-webkit-box-orient: vertical` as well so it is supported, Firefox supports the webkit prefix as well: https://developer.mozilla.org/en-US/docs/Web/CSS/box-orient

**Comment by mgol** [comment]
@ai Firefox has implemented [`webkit-line-clamp`](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp) in v68 (currently in beta). To use it, you have to set `-webkit-box-orient: vertical` as well so it is supported, Firefox supports the webkit prefix as well: https://developer.mozilla.org/en-US/docs/Web/CSS/box-orient

**Comment by mgol** [comment]
See the news with more details here: https://webplatform.news/issues/2019-05-17

**Comment by shivam2407** [comment]
> Use:
> 
> ```css
>   /* autoprefixer ignore next */
>   -webkit-box-orient: vertical;
> ```
> 
> Does this code works in Firefox and other non-webkit browsers?

```
/* autoprefixer: ignore next */
-webkit-box-orient: vertical;
```
Above code should worked for me.

**Comment by ai** [comment]
Fixed 69ed8a0

**Comment by ai** [comment]
Released in 9.6.1.
